extension/options.html

```diff
@@ -120,6 +120,56 @@
 
     </section>
 
+    <section class="panel">
+        <h2>Site profiles</h2>
+        <table id="saved_site_profiles">
+            <thead>
+                <tr>
+                    <th>Domain</th>
+                    <th>Change domain to</th>
+                    <th>Append string</th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody></tbody>
+        </table>
+    </section>
+
+    <section class="panel">
+        <form id="newSiteProfile">
+            <h2>Create new site profile</h2>
+
```

Comment:
I feel like a little explanation could be added here just about what site profiles are for - something like:

> Configure additional settings for passwords on a per domain basis.